### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by maskalek** [Biarity/Sieve#189] [General]
Is this project dead? Any alternatives/forks?

**Comment by ITDancer13** [comment]
Unfortunately I was not able to jump into the project as I wanted. But I still want to keep it running if some others are still with me. I try to release version 3 a soon as possible.

**Reply by sathishsuresh04** [reply]
Any plan for v3 release sooner?

**Reply by soroshsabz** [reply]
Any update? or plan to add more collaborators?

**Reply by soroshsabz** [reply]
@ITDancer13 @hasanmanzak

**Comment by hasanmanzak** [comment]
No actually. I'm also like @ITDancer13, could not be able to find the time and put the work down.. I'm gonna to be more active within the project in the upcoming future!

**Reply by LoveShakeDev** [reply]
How soon is the upcoming future? It's been more than a year since you made this comment.

**Reply by soroshsabz** [reply]
Can you add some additional collaborator to help more development on this repo?

**Reply by soroshsabz** [reply]
@hasanmanzak

**Reply by sathishsuresh04** [reply]
This project is really a good one, it would be great if we keep this project alive  by adding additional collaborators :-)

**Comment by AfzalSohaib** [comment]
Gridify appears to be an alternative but I have not tried it myself.

**Reply by sathishsuresh04** [reply]
> Gridify appears to be an alternative but I have not tried it myself.

Any idea how to add Custom sort methods at Gridify similar to that one in Sieve
